app/frontend/components/pages/delegations/shelleyBalances.tsx

```diff
@@ -66,11 +65,7 @@ const shelleyBalances = ({
         )}
       </div>
       {!!rewardsAccountBalance && (
-        <button
-          disabled={calculatingDelegationFee}
-          className="button secondary balance withdraw"
-          onClick={withdrawRewards}
-        >
+        <button className="button secondary balance withdraw" onClick={withdrawRewards}>
```

Comment:
didn't the disabling of the button while delegation fees are being computed serve as a protection against repeated triggering of the logic? Shouldn't we debounce it then?

Reply:
Well, the `calculatingDelegationFee` is not related to `withdrawRewards`, but to calculation of delegation fees trigers immediately when user lands on this screen. Now they are not sharing transaction plans, so this is why it was removed. Do you see there some other potential problems?

Reply:
And the repeated protection is not there, but also was not there, as `calculatingDelegationFee` is not set when `withdrawRewards` is called.